gamedata/scavengers/unitdef_changes.lua

```diff
@@ -97,7 +96,7 @@ customDefs.corcom = {
 	stealth = false,
 	workertime = 200,				-- can get multiplied in unitdef_post
 	customparams = {
-		iscommander = 'nil',		-- since you cant actually remove parameters normally, it will do it when you set string: 'nil' as value
+		iscommander = nil,
```

Comment:
But things are `nil` by default, maybe the whole entry could be removed?

Reply:
These unitdef tables contain only changes to be merged with the original unitdef table, so it is overwriting the existing `iscommander = true` entry

Reply:
Yes, I mean that it may not be working the way you anticipate. Check what happens in the following cases.

1)
```
customparams = { }
```
I think your assumption would be that this doesn't remove the `iscommander` field but I think it should remove it correctly (which is why I propose removing the entry here).

2)
```
customparams = {
   iscommander = nil,
   test_value_for_replacement = "bla",
}
```
I think your assumption here would be that `iscommander` is removed but I claim it won't be.